node/go.mod

```diff
@@ -19,30 +18,30 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/improbable-eng/grpc-web v0.15.0
 	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/libp2p/go-libp2p v0.22.0
-	github.com/libp2p/go-libp2p-kad-dht v0.18.0
-	github.com/libp2p/go-libp2p-pubsub v0.8.0
+	github.com/libp2p/go-libp2p v0.27.0
```

Comment:
Why did you bump libp2p? This might have unintended side-effects.

Reply:
libp2p imports quic-go. quic-go < [v0.32.0](https://github.com/quic-go/quic-go/releases/tag/v0.32.0) doesn't support go 1.20. 
libp2p needs to be >= [v0.26.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.26.0) to import quic-go >= v0.32.0. 
Since we need to upgrade libp2p anyways to >=v0.26.0, might as well upgrade to the latest (v0.27.0)